src/Executables/FindHorizons/FindHorizons.hpp

```diff
@@ -0,0 +1,362 @@
+    get<gr::Tags::SpatialMetric<Dim, Frame::Inertial, DataVector>>(
+        *target_vars) = spatial_metric;
+    get<gr::Tags::ExtrinsicCurvature<Dim, Frame::Inertial, DataVector>>(
+        *target_vars) = ext_curvature;
+    auto& inv_spatial_metric =
+        get<gr::Tags::InverseSpatialMetric<Dim, Frame::Inertial, DataVector>>(
+            *target_vars);
+    Scalar<DataVector> unused_det{mesh.number_of_grid_points()};
+    determinant_and_inverse(make_not_null(&unused_det),
+                            make_not_null(&inv_spatial_metric), spatial_metric);
+    const auto deriv_spatial_metric =
+        ::partial_derivative(spatial_metric, mesh, inv_jacobian);
+    auto& spatial_christoffel_second_kind =
+        get<gr::Tags::SpatialChristoffelSecondKind<Dim, Frame::Inertial,
+                                                   DataVector>>(*target_vars);
+    gr::christoffel_second_kind(make_not_null(&spatial_christoffel_second_kind),
+                                deriv_spatial_metric, inv_spatial_metric);
+    const auto deriv_spatial_christoffel_second_kind = ::partial_derivative(
+        spatial_christoffel_second_kind, mesh, inv_jacobian);
+    auto& spatial_ricci =
+        get<gr::Tags::SpatialRicci<Dim, Frame::Inertial>>(*target_vars);
+    gr::ricci_tensor(make_not_null(&spatial_ricci),
+                     spatial_christoffel_second_kind,
+                     deriv_spatial_christoffel_second_kind);
+  }
+};
+
+template <size_t Dim>
+struct AhA {
+ private:
+  using tags_to_observe = tmpl::list<
+      StrahlkorperGr::Tags::AreaCompute<Frame::Inertial>,
+      StrahlkorperGr::Tags::IrreducibleMassCompute<Frame::Inertial>,
+      StrahlkorperTags::MaxRicciScalarCompute,
+      StrahlkorperTags::MinRicciScalarCompute,
+      StrahlkorperGr::Tags::ChristodoulouMassCompute<Frame::Inertial>,
+      StrahlkorperGr::Tags::DimensionlessSpinMagnitudeCompute<Frame::Inertial>>;
+
+ public:
+  using temporal_id = ::Tags::Time;
+  using compute_target_points =
+      intrp::TargetPoints::ApparentHorizon<AhA, Frame::Inertial>;
+  using post_interpolation_callback =
+      intrp::callbacks::FindApparentHorizon<AhA, Frame::Inertial>;
+  using horizon_find_failure_callback =
+      intrp::callbacks::ErrorOnFailedApparentHorizon;
+  using post_horizon_find_callback =
+      intrp::callbacks::ObserveTimeSeriesOnSurface<tags_to_observe, AhA, AhA>;
+
+  using vars_to_interpolate_to_target =
+      tmpl::list<gr::Tags::SpatialMetric<Dim, Frame::Inertial>,
+                 gr::Tags::InverseSpatialMetric<Dim, Frame::Inertial>,
+                 gr::Tags::ExtrinsicCurvature<Dim, Frame::Inertial>,
+                 gr::Tags::SpatialChristoffelSecondKind<Dim, Frame::Inertial>,
+                 gr::Tags::SpatialRicci<Dim, Frame::Inertial>>;
+  using compute_vars_to_interpolate = ComputeHorizonVolumeQuantities<Dim>;
+  using compute_items_on_target = tmpl::append<
+      tmpl::list<
+          StrahlkorperGr::Tags::AreaElementCompute<Frame::Inertial>,
```

Comment:
Not requesting a change, but this monster list of tags now appears in at least 3 places (here, BBH exec, and GH with horizon exec). Maybe we should consider defining this list (and potentially the ones above as well) in their own file in `ApparentHorizons` so anything wanting to do a horizon find can just use a common list of tags.